dao/src/main/java/com/epam/esm/dao/impl/sorting/SortParameterInserter.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.esm.dao.impl.sorting;
+
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.From;
+import java.util.List;
+
+public interface SortParameterInserter {
+    default <R, T extends Enum<T>, X, Z> void addSortingParams (
```

Comment:
It seems to me that the default methods are not being used correctly, can we take a different approach for this method?